pages/api/bounties/index.ts

```
@@ -0,0 +1,36 @@
+
+import { NextApiRequest, NextApiResponse } from 'next';
+import nc from 'next-connect';
+import { Bounty, Prisma, Space } from '@prisma/client';
+import { prisma } from 'db';
+import { onError, onNoMatch, requireUser } from 'lib/middleware';
+import { withSessionRoute } from 'lib/session/withSession';
+import { gettingStartedPageContent } from 'seedData';
+
+const handler = nc<NextApiRequest, NextApiResponse>({ onError, onNoMatch });
+
+handler.use(requireUser).get(getBounties).post(createBounty);
+
+async function getBounties (req: NextApiRequest, res: NextApiResponse<Bounty[]>) {
+
+  const { workspaceId } = req.query;
+
+  const bountyListQuery: Prisma.BountyFindManyArgs = workspaceId ? {
```

this route should honestly just throw a 400 if spaceId is not defined.

---

Added this [here](https://github.com/charmverse/app.charmverse.io/pull/102/commits/58c49e105658ba65a0b46a2ada26185c408523c1)